examples/snippets/dataset/_profile/create_retrieve_profile.py

```diff
@@ -0,0 +1,23 @@
+"""
+An example script to create and/or retrieve a dataset profile
+"""
+
+import tamr_toolbox as tbox
+from requests import HTTPError
+
+# load example config
+my_config = tbox.utils.config.from_yaml("examples/resources/conf/migrate_dataset.config.yaml")
+
+# Create the source tamr client
+client = tbox.utils.client.create(**my_config["source_migration_instance"])
+
+# Get dataset from Tamr instance
+dataset_name = "<your dataset name>"
+dataset = client.datasets.by_name(dataset_name)
+
+try:
+    profile = tbox.dataset.get_profile(dataset)
+except HTTPError:
```

Comment:
Does the exception need to be handled here if it's already part of the get_profile definition?

Reply:
So this is a design question. Following @skalish 's suggestion during the last office hours, I designed the default to not create a profile and raise an error instead. Should I just put this in the comments for the example instead of a full try except clause?